ruff.toml

```
@@ -25,7 +13,10 @@ extend-exclude = [
     "pydispatch",
 ]
 
-[tool.ruff.lint]
+line-length = 88
```

```suggestion
line-length = 79
```
Haben wir bei black und flake8 nicht auch nur 80 Zeichen?